src/it/java/io/ecocode/java/integration/tests/profile/RuleMetadata.java

```diff
@@ -0,0 +1,40 @@
+package io.ecocode.java.integration.tests.profile;
+
+public class RuleMetadata {
+	private String key;
+	private String type;
+	private String defaultSeverity;
+
+	public String getKey() {
```

Comment:
lombok ?

Reply:
Lombok can make it more complex for other contributors to configure some IDEs. I prefer to avoid it.